src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -454,11 +454,16 @@ private function buildTableRows($rows)
      * @param int   $line
      *
      * @return array
+     *
+     * @throws InvalidArgumentException
      */
     private function fillNextRows(array $rows, $line)
     {
         $unmergedRows = array();
         foreach ($rows[$line] as $column => $cell) {
+            if (null !== $cell && !is_string($cell) && !$cell instanceof TableCell && !is_scalar($cell)) {
+                throw new InvalidArgumentException('The cell should always be a string, a TableCell or a scalar, array given.');
```

Comment:
`gttype()` ?

Reply:
`array given` is too strict. It could be anything. Using `gettype()` would indeed give something more specific.